pkg/cmd/util/downloadrequest/downloadrequest.go

```diff
@@ -125,6 +143,16 @@ func download(
 		}
 		caPool.AppendCertsFromPEM(caCert)
 	}
+	if len(caCertByteString) > 0 {
```

Comment:
1. If there's error opening the `caCertFile` shall we fallback to BSL cert?
2. Seems these two "if" chunks can be combined so the `caPool` can be reused.